addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -157,7 +157,7 @@ class ThreadView extends Component {
         if (!this.threadView.threadCache) {
             return;
```

Comment:
condition on thread cache can be moved in model (commit: Move onClickRetryLoadMessages)